### ISSUE TITLE
battery limits bug?

### ISSUE THREAD

**Issue by 687766616e** [alainesp/HashSuiteDroid#2]
I set the "battery limits" to "stop if level<5% or t>70°c", but it stops at 45 degrees, no matter how many 

_從我的 MI6，使用 [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github) 發送。_

**Comment by alainesp** [comment]
Hi Huitc.

On Mar 25, 2018 11:04 AM, "huitc" <notifications@github.com> wrote:

I set the "battery limits" to "stop if level<5% or t>70°c", but it stops at
45 degrees, no matter how many

Android had an option to check if the battery is OVERHEATING, whatever that
means for a specific device. We check that besides the two parameters you
discuss. So is possible this is the cause of your problem.

Hope this helps,
Alain

**Comment by 687766616e** [comment]
Will this APP continue to be updated?

**Comment by alainesp** [comment]
Yes.

**Comment by solardiz** [comment]
Was this issue possibly about the bug fixed in 1.3.3 with the below change? If so, should we perhaps close it as assumed-fixed?

```diff
	public static int getBatteryMaxTemperature()
	{
-		return params.getInt(pref_key_battery_limit, 50) >> 8;
+		return params.getInt(pref_key_battery_limit, (50<<8)) >> 8;
	}
```

**Comment by alainesp** [comment]
Yes, it is resolved in version 1.3.3

**Comment by alainesp** [comment]
On Thu, Apr 23, 2020 at 6:10 AM Solar Designer <notifications@github.com>
wrote:

> Was this issue possibly about the bug fixed in 1.3.3 with the below
> change? If so, should we perhaps close it as assumed-fixed?
>
> 	public static int getBatteryMaxTemperature()
> 	{-		return params.getInt(pref_key_battery_limit, 50) >> 8;+		return params.getInt(pref_key_battery_limit, (50<<8)) >> 8;
> 	}
>
> Yes. I close the Github issue.

Regards,
Alain
